### PR TITLE
Added Devuan

### PR DESCRIPTION
**Pull request by thierrybo**
Added Devuan to the list  of available distributions.

### CODE REVIEW

File: quickget [hunk 1]
```diff
@@ -1643,6 +1693,8 @@ if [ -n "${2}" ]; then
         fi
         VM_PATH="${OS}-${RELEASE}-${FREEDOM}"
         get_debian  "${FREEDOM}"
+    elif [ "${OS}" == "devuan" ]; then
+         get_devuan
```
Comment (zen0bit) [inline quickget:1697]: one less space here, otherwise excellent i think...

**Review by zen0bit — APPROVED** [review]

File: quickget (hunk 1 above)
Comment (thierrybo) [reply quickget]: done

**Comment by flexiondotorg** [comment]
Thanks for the pull request that adds Devuan. We've change some conditionals from `if`/`then` to case. Please can your refactor the pull request some it merged cleanly? 🙂

**Comment by thierrybo** [comment]
**devaun** instead of devuan
```
 30 quickget
@@ -194,6 +194,7 @@ function os_support() {
    arcolinux \
    cachyos \
    debian \
    **devaun** \
```

**Pull request closed without merging**